waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -162,7 +162,7 @@ proc hasPeers*(pm: PeerManager, proto: string): bool =
   # Returns `true` if manager has any peers for the specified protocol
   pm.peers.anyIt(it.protos.contains(proto))
 
-proc addPeer*(pm: PeerManager, peerInfo: PeerInfo, proto: string) {.raises: [Defect, Exception]} =
```

Comment:
Could you please explain why the `raises` pragma is removed in here?

Reply:
I think that is because of the `{.push raises: [Defect].}` https://github.com/status-im/nim-waku/blob/37c5a6eb222921375a2f9bcedc9133b2c4b1b705/waku/v2/node/peer_manager/peer_manager.nim#L1` at the top of the file

Reply:
True, but also because the underlying exception is now caught and handled elsewhere in the `peer_manager` and therefore not raised by `addPeer`.

Reply:
Thanks for the clarification!